apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -172,6 +172,34 @@ public object Clone()
         /// <inheritdoc />
         public override int ExecuteNonQuery() => ExecuteNonQueryAsync(_synchronousCancellationTokenSource.Token).ResultWithUnwrappedExceptions();
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public Task<DbDataReader> ExecuteReaderAsync(TimestampBound singleUseReadSettings)
+        {
+            return ExecuteReaderAsync(singleUseReadSettings, CancellationToken.None);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public async Task<DbDataReader> ExecuteReaderAsync(TimestampBound singleUseReadSettings, 
+            CancellationToken cancellationToken)
```

Comment:
We use this in many other places in the codebase. It is called out as a recommended pattern [here](https://blogs.msdn.microsoft.com/andrewarnottms/2014/03/19/recommended-patterns-for-cancellationtoken/). I think it should be safe to depend on the assumption that `default(CancellationToken) ` and `CancellationToken.None` are equivalent, which is documented [here](https://msdn.microsoft.com/en-us/library/system.threading.cancellationtoken.none.aspx)